faunadb/streams/client.py

```diff
@@ -0,0 +1,115 @@
+from time import time
+
+try:
+    #python2
+    from urllib import urlencode
+except ImportError:
+    #python3
+    from urllib.parse import urlencode
+
+from hyper import HTTP20Connection
+from faunadb._json import to_json, parse_json_or_none
+from faunadb.request_result import RequestResult
+from .events import parse_stream_request_result_or_none, Error
+from .errors import StreamError
+
+VALID_FIELDS = {"ref", "ts", "diff", "prev", "document", "action"}
```

Comment:
This has to be updated with the new events format. Top level fields are now: `action`, `document`, `prev`, and `diff`.

Reply:
@erickpintor missed the github notification for this. Fixed now